### PR TITLE
Modified answer.txt file

### PR DESCRIPTION
**Pull request by Nagarjun1504**

### CODE REVIEW

File: answer.txt [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
 	*.ear
 	*.nar
 	target/
 
-	#Intellij Specific :
+	#Intellij Specific : 
 	.idea/
 	.idea_modules/
-	*.iml
+	*.iml
```
Comment (chandankj399) [inline answer.txt:18]: This looks good

File: answer.txt [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
+	This doesnt create any merge conflict and the whole process is called as fast-forward. 
 	
-3. Why should you have different branches?
+3. Why should you have different branches? 
 	
 	In a project, various developers implement different User Stories/Features. Each developer will need to create a 
 	different branch so that the new feature being implemented does not conflict with the other feature being implemented. 
 	Creating a branch will clone the base branch and represent a copy of the base branch. At the end of the implementation, 
-	developers create a pull request and resolve the conflicts(if any) and push it to the base branch (often called as Master).
+	developers create a pull request and resolve the conflicts(if any) and push it to the base branch (often called as Master).
```
Comment (chandankj399) [inline answer.txt:35]: This covers the answer but one more reason for using git branches could have been mentioned

File: answer.txt [hunk 3]
```diff
@@ -1,36 +1,36 @@
...
 	Creating a branch will clone the base branch and represent a copy of the base branch. At the end of the implementation, 
-	developers create a pull request and resolve the conflicts(if any) and push it to the base branch (often called as Master).
+	developers create a pull request and resolve the conflicts(if any) and push it to the base branch (often called as Master). 
 	
-4. In a project how many remote Git repositories do you usually have?
+4. In a project how many remote Git repositories do you usually have? 
 	
-	One remote git repository
\ No newline at end of file
+	One remote git repository
```
Comment (chandankj399) [inline answer.txt:38]: Whats the disadvantages of having multiple remote git repos?

**Review by chandankj399 — COMMENTED** [review]
Answers look good for the most part

File: answer.txt (hunk 2 above)
Comment (Nagarjun1504) [reply answer.txt:35]: I have mentioned another advantage of using branches.

File: answer.txt (hunk 3 above)
Comment (Nagarjun1504) [reply answer.txt:38]: Maintainability will be difficult in case of multiple remote repositories.

**Review by chandankj399 — APPROVED** [review]
Changes Approved

**Pull request merged**